modules/core/role/modules/account_admin/config/install/user.role.farm_account_admin.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  enforced:
+    module:
+      - farm_role_account_admin
+  module:
+    - farm_role
+id: farm_account_admin
+label: 'Account Admin'
+weight: 1
+is_admin: false
+permissions:
+  - 'administer farm settings'
+  - 'administer users'
+third_party_settings:
+  farm_role:
+    access:
+      config: true
+      entity:
+        view all: true
+        create all: true
+        update all: true
+        delete all: true
```

Comment:
I wonder if it makes sense to grant these write permissions?

Without them, it opens up the possibility of having a user Alice who operates in an administrative capacity to add/modify other users, but doesn't interact with the farm data itself. Obviously Alice could modify her own account to also add the `farm_manager` role, but not having these permissions by default could act like a safety net where she would have to go out of her way to act outside that administrative capacity.

On the other hand, as-is this maintains the superset relationship between each successively more permissive role. i.e. `account_admin > manager > worker > viewer`. That's probably a bit easier to document/explain, but I personally see it as a bit unintuitive that account admin would implicitly give create/edit permissions on assets.

Reply:
Giving this some more thought: I suppose I'm actually a bit in favor of limiting the permissions - it's nice to keep the roles focused. and it helps to demonstrate that roles "layer" on one another. I'll push a quick commit that makes this change and we can roll it back if @paul121 or other disagree. :-)

Reply:
https://github.com/farmOS/farmOS/pull/714/commits/e3228f08ac7b82015adb1634dac17fe154bab211

Note: this is leaving `config: true`, but all entity permissions to `false`. This is because we're also using this role for the `administer farm settings` permission, which sort shines new light on a consideration I mentioned earlier:

> this role ALSO adds the `administer farm settings` permission. This is a carry over from Farmier, and it sort of stretches the purpose of this module